### PR TITLE
chore: add release notes for 1.28.0

### PR DESCRIPTION
**Pull request by vwong**

### CODE REVIEW

File: scripts/release/release-notes-onprem.sh [hunk 1]
```diff
@@ -60,7 +60,7 @@ echo "Fetching application..."
 if [ -d $PACTFLOW_APPLICATION_DIR ]; then
     rm -rf $PACTFLOW_APPLICATION_DIR
 fi
-git clone --shallow-since=$(date -v -2m "+%Y-%m-%d") git@github.com:pactflow/pactflow-application.git $PACTFLOW_APPLICATION_DIR >/dev/null 2>&1
+git clone --shallow-since=$(date -v -3m "+%Y-%m-%d") git@github.com:pactflow/pactflow-application.git $PACTFLOW_APPLICATION_DIR >/dev/null 2>&1
```
Comment (vwong) [inline scripts/release/release-notes-onprem.sh:63]: Lol, we need to release more often!

File: website/docs/docs/on-premises/environment-variables.md [hunk 2]
```diff
@@ -292,13 +292,6 @@ The URL to which the authentication request should be sent. This endpoint is on
 **Required:** if PACTFLOW_SAML_IDP_METADATA_URL is not set<br/>
 **More information:** https://github.com/omniauth/omniauth-saml#options<br/>
 
-### PACTFLOW_SAML_IDP_ENTITY_ID
-
-The ID by which this IDP is known to PactFlow
-
-**Required:** if PACTFLOW_SAML_IDP_METADATA_URL is not set<br/>
-**More information:** https://github.com/omniauth/omniauth-saml#options<br/>
-
```
Comment (vwong) [inline website/docs/docs/on-premises/environment-variables.md:301]: This got deleted automatically. Is this intended?

File: website/docs/docs/on-premises/releases/1.28.0.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+
+## Release date
+
+28 November 2023
+
+## Features
+
+* Add endpoint to list branches
+* Removed "generate project" links
+* Onprem customers can now specify additional sources to add to the Content-Security-Policy
```
Comment (bethesque) [inline website/docs/docs/on-premises/releases/1.28.0.md:13]: These notes are only for onprem, so you can remove the qualification at the start of the sentence. It should have a similar "voice" to the other features. eg.

"Allow additional sources to be added to the Content-Security-Policy"

**Comment by vwong** [comment]
Is that failing build expected?

**Comment by bethesque** [comment]
I would say, not expected, but it should not stop you merging. `algoliasearch.helpers.AlgoliaException: You have exceeded your Record quota. You’ll need to change your plan for more capacity, or delete records.`

**Comment by bethesque** [comment]
I have docs to add about `PACTFLOW_ALLOW_DANGEROUS_CONTRACT_MODIFICATION`, but I can add them after your release.

**Pull request merged**